libs/wingsdk/src/target-tf-aws/bucket.inflight.ts

```diff
@@ -44,6 +50,37 @@ export class BucketClient implements IBucketClient {
     return JSON.parse(await this.get(key));
   }
 
+  private async exists(key: string): Promise<boolean> {
+    const command = new ListObjectsCommand({
+      Bucket: this.bucketName,
+      Prefix: key,
+      MaxKeys: 1,
+    });
+    const resp: ListObjectsCommandOutput = await this.s3Client.send(command);
+    return !!(resp.Contents && resp.Contents.length > 0);
```

Comment:
I see what you are saying, but still will have to add the `MaxKeys: 1` option, I can create another private list method, but then it will just be a wrapper to the `ListObjectsCommand` and `this.s3Client.send(command)` together, and be used in both of the functions, what is the preferred way to do it here?

Reply:
hmm, I think because we are searching for a specific key that we dont have to worry about `MaxKeys: 1` all we really care about is that the response when searching for 'some/object/key` returns at least the 1 match.

Reply:
but if we'll omit the `MaxKeys: 1`, if we have 20,000 items under some prefix, it will make the `public_url` method run potentially for a longer time than if we have only 10...

Reply:
Yes, I guess the MaxKeys default is 1,000 but we should probably still limit it. Maybe just add an optional param to the list method that might be enough rather than doing a wrapper.

Reply:
I tend to keep it separated for now, this way if we'll ever add more params to the `list` function we won't have to change the `exists` function as well, it keeps us more flexible :) wdyt?